test/runtests.jl

```diff
@@ -59,6 +60,13 @@ const GROUP = get(ENV, "GROUP", "")
 include("test_utils.jl")
 
 @testset "KernelFunctions" begin
+    if GROUP == "" || GROUP == "Aqua"
+        @testset "Aqua" begin
+            Aqua.test_all(KernelFunctions; ambiguities=false)
+            Aqua.test_ambiguities(KernelFunctions; recursive=false)
```

Comment:
Should it be
```suggestion
            Aqua.test_ambiguities([KernelFunctions, Base, Core]; recursive=false)
```
to be consistent with what `test_all` would do? (I'm not quite clear why you don't want to do it recursively, though.)